test/Microsoft.DotNet.Docker.Tests/ImageTests.cs

```diff
@@ -25,41 +26,56 @@ public ImageTests(ITestOutputHelper output)
 
         public static IEnumerable<object[]> GetVerifyImagesData()
         {
-            List<ImageDescriptor> testData = new List<ImageDescriptor>
-            {
-                new ImageDescriptor {DotNetCoreVersion = "1.0", SdkVersion = "1.1"},
-                new ImageDescriptor {DotNetCoreVersion = "1.1", RuntimeDepsVersion = "1.0"},
-                new ImageDescriptor {DotNetCoreVersion = "2.0"},
-                new ImageDescriptor {DotNetCoreVersion = "2.1", RuntimeDepsVersion = "2.0"},
-            };
+            List<ImageDescriptor> testData;
 
             if (DockerHelper.IsLinuxContainerModeEnabled)
             {
-                testData.AddRange(new List<ImageDescriptor>
+                testData = new List<ImageDescriptor>
+                {
+                    new ImageDescriptor { DotNetCoreVersion = "1.0", SdkVersion = "1.1" },
+                    new ImageDescriptor { DotNetCoreVersion = "1.1", RuntimeDepsVersion = "1.0" },
+                    new ImageDescriptor { DotNetCoreVersion = "2.0" },
+                    new ImageDescriptor { DotNetCoreVersion = "2.0", OsVariant = "jessie" },
+                    new ImageDescriptor
+                    {
+                        DotNetCoreVersion = "2.0",
+                        OsVariant = "stretch",
```

Comment:
How about an enumeration? Benefit is allows user to avoid spell errors such `strecth` or `jesse`. Similarly for PlatformOS.

Reply:
I have some thoughts on how to clean this up in general.  One problem I have with using an enum is that the length increases which doesn't allow for a single line definition which really hurts the readability.  I will make a follow up PR to address this.

Reply:
I knew you would :) Thanks!